.github/workflows/sonarcloud.yml

```
@@ -46,7 +46,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
+        uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)